src/js/populationSlider.ts

```diff
@@ -13,28 +13,26 @@ const draw = (
   rightIndex: number
 ): void => {
   // We dynamically change the sliders so that they cannot extend past each other.
-  const cross = rightIndex - 0.5 >= leftIndex ? rightIndex - 0.5 : rightIndex; // a single interval that min and max sliders overlap
-  const extend = leftIndex + 1 == rightIndex; // if sliders are close and within 1 step can overlap
-  const newLeftMax = extend ? cross + 0.5 : cross;
-  const newRightMin = cross;
+  const inBetween = (rightIndex - leftIndex) / 2;
+  const newLeftMax = leftIndex + inBetween;
+  const newRightMin = rightIndex - inBetween;
 
   sliders.left.setAttribute("max", newLeftMax.toString());
   sliders.right.setAttribute("min", newRightMin.toString());
+  sliders.left.setAttribute("value", leftIndex.toString());
+  sliders.right.setAttribute("value", rightIndex.toString());
 
   const intervalSizePx =
-    sliders.controls.offsetWidth / POPULATION_INTERVALS.length;
+    (sliders.controls.offsetWidth + THUMBSIZE / 2) /
+    POPULATION_INTERVALS.length;
```

Comment:
`THUMBSIZE / 2` is a guess. The slider wasn't taking up the whole div, and it looked like it was off by half a header.